app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,87 @@
+// import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from 'App/Models/Profile';
+import { schema, rules } from '@ioc:Adonis/Core/Validator';
+export default class ProfilesController {
+    public async addProfile({ auth, request, response }) {
+        await auth.use('api').check()
+        if (auth.use('api').isLoggedIn) {
```

Comment:
```ts
await auth.use('api').check()
if (auth.use('api').isLoggedIn) {
```

I don't think we would need to check this since we already have checked this using the `auth` middleware